tools/test/audio/process_test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2020 Intel Corporation. All rights reserved.
+
+set -e
+
+# This process_test.sh is shell script wrapper for process_test.m
+# This needs octave octave-signal octave-io packages.
+
+main ()
+{
+	echo "process_test, " Component="$1", "$2", "$3", "$4", "$5"
+	octave -q --eval "process_test('$1', $2, $3, $4, $5)"
```

Comment:
A trick to return error code:

```suggestion
	octave -q --eval "[n_fail, n_pass]=process_test('$1', $2, $3, $4, $5);exit(n_fail + !n_pass)"
```